CONTRIBUTING.md

```diff
@@ -8,49 +8,39 @@ to the rules described here, but by following the instructions below you
 should have a much easier time getting your work merged with the upstream
 project.
 
-* Test Your Code
+## Test Your Code Using Existing Tests
 
-There are two possible tests you can run to verify your code.  The first test
-is used to check the formatting and coding style of your changes, you can run
-the test with the following command:
+There are three possible tests you can run to verify your code.  The first
+test is used to check the formatting and coding style of your changes, you
+can run the test with the following command:
 
 	# make check-syntax
 
 ... if there are any problems with your changes a diff/patch will be shown
 which indicates the problems and how to fix them.
 
-The second possible test is used to ensure the sanity of your code changes
-and to test these changes against the included tests.  You can run the test
-with the following command:
+The second possible test is used to ensure that the different internal syscall
+tables are consistent and to test your changes against the automated test
+suite.  You can run the test with the following command:
 
 	# make check
 
-... if there are any faults or errors they will be displayed.
+... if there are any faults or errors they will be displayed; beware that the
+tests can run for some time and produce a lot of output.
 
-* Generate the Patch(es)
+## Add New Tests for New Functionality
 
-Depending on how you decided to work with the libseccomp code base and what
-tools you are using there are different ways to generate your patch(es).
-However, regardless of what tools you use, you should always generate your
-patches using the "unified" diff/patch format and the patches should always
-apply to the libseccomp source tree using the following command from the top
-directory of the libseccomp sources:
+The libseccomp code includes a fairly extensive test suite and any submissions
```

Comment:
Hmm, the golang tests are actually pretty meager compares to the C/Python tests.  Perhaps change this to read "Any submissions which add functionality ..."?